component/Buttons/button.css

```diff
@@ -0,0 +1,60 @@
+.button{
+    background-color:var(--primary-color);
+    color:var(--white-color);
+    border-radius:3px;
+    border:0px solid var(--white-color);
+    font-size: 1.8rem;
+    font-weight: 400;
+    margin:2rem 2rem;
+    width:12vw;
```

Comment:
Don’t use vw and vh here. Try checking this in mobile. The button will get very small.

Reply:
should i use rem in place of them.

Reply:
done

Reply:
done